front/app/components/EsriMap/index.tsx

```diff
@@ -214,13 +241,48 @@ const EsriMap = ({
   // Note: This data is dynamic and may change.
   useEffect(() => {
     // Add any map layers which were passed in
+
+    // Handle when we're using a webmap
+    webMap?.when(() => {
+      if (webMap && layers) {
+        // If there are any Web Map reference layers, re-order so they are below other layers created in our application
```

Comment:
Esri Web Maps can have reference layers sometimes (E.g. A layer with text labels for streets/neighborhoods, etc.) which always sit on top of everything else on the map. However, in our application we still want our layers (like the Idea pins) to sit on top of the reference layer. So we just re-order them when we handle layers here.